ledger/acctupdates.go

```diff
@@ -586,100 +586,6 @@ func (au *accountUpdates) LookupWithoutRewards(rnd basics.Round, addr basics.Add
 	return
 }
 
-// ListAssets lists the assets by their asset index, limiting to the first maxResults
-func (au *accountUpdates) ListAssets(maxAssetIdx basics.AssetIndex, maxResults uint64) ([]basics.CreatableLocator, error) {
-	return au.listCreatables(basics.CreatableIndex(maxAssetIdx), maxResults, basics.AssetCreatable)
-}
-
-// ListApplications lists the application by their app index, limiting to the first maxResults
-func (au *accountUpdates) ListApplications(maxAppIdx basics.AppIndex, maxResults uint64) ([]basics.CreatableLocator, error) {
-	return au.listCreatables(basics.CreatableIndex(maxAppIdx), maxResults, basics.AppCreatable)
-}
-
-// listCreatables lists the application/asset by their app/asset index, limiting to the first maxResults
-func (au *accountUpdates) listCreatables(maxCreatableIdx basics.CreatableIndex, maxResults uint64, ctype basics.CreatableType) ([]basics.CreatableLocator, error) {
-	au.accountsMu.RLock()
-	for {
-		currentDbRound := au.cachedDBRound
-		currentDeltaLen := len(au.deltas)
-		// Sort indices for creatables that have been created/deleted. If this
-		// turns out to be too inefficient, we could keep around a heap of
-		// created/deleted asset indices in memory.
-		keys := make([]basics.CreatableIndex, 0, len(au.creatables))
-		for cidx, delta := range au.creatables {
-			if delta.Ctype != ctype {
-				continue
-			}
-			if cidx <= maxCreatableIdx {
-				keys = append(keys, cidx)
-			}
-		}
-		sort.Slice(keys, func(i, j int) bool { return keys[i] > keys[j] })
-
-		// Check for creatables that haven't been synced to disk yet.
-		unsyncedCreatables := make([]basics.CreatableLocator, 0, len(keys))
-		deletedCreatables := make(map[basics.CreatableIndex]bool, len(keys))
-		for _, cidx := range keys {
-			delta := au.creatables[cidx]
-			if delta.Created {
-				// Created but only exists in memory
-				unsyncedCreatables = append(unsyncedCreatables, basics.CreatableLocator{
-					Type:    delta.Ctype,
-					Index:   cidx,
-					Creator: delta.Creator,
-				})
-			} else {
-				// Mark deleted creatables for exclusion from the results set
-				deletedCreatables[cidx] = true
-			}
-		}
-
-		au.accountsMu.RUnlock()
-
-		// Check in-memory created creatables, which will always be newer than anything
-		// in the database
-		if uint64(len(unsyncedCreatables)) >= maxResults {
-			return unsyncedCreatables[:maxResults], nil
-		}
-		res := unsyncedCreatables
-
-		// Fetch up to maxResults - len(res) + len(deletedCreatables) from the database,
-		// so we have enough extras in case creatables were deleted
-		numToFetch := maxResults - uint64(len(res)) + uint64(len(deletedCreatables))
-		dbResults, dbRound, err := au.accountsq.ListCreatables(maxCreatableIdx, numToFetch, ctype)
```

Comment:
forgot to remove `func (qs *accountsDbQueries) ListCreatables` in sql.go and `listCreatablesStmt` there as well

Reply:
Thanks.
https://github.com/algorand/go-algorand/pull/5495